timbermill-client/src/test/java/com/datorama/timbermill/TimbermillStressTest.java

```diff
@@ -0,0 +1,185 @@
+package com.datorama.timbermill;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import com.datorama.timbermill.annotation.TimberLog;
+import com.datorama.timbermill.pipe.TimbermillServerOutputPipe;
+import com.datorama.timbermill.unit.Event;
+import com.datorama.timbermill.unit.LogParams;
+import com.datorama.timbermill.unit.Task;
+import com.google.common.collect.Lists;
+
+import static org.junit.Assert.*;
+
+public class TimbermillStressTest extends TimberLogTest{
+
+    private static final int NUM_OF_TASKS = 10000;
+    private static final int NUM_OF_THREADS = 10;
+    private static final int NUMBER_OF_PARENTS = 30;
+    private static final String CTX = "ctx";
+    private static ExecutorService executorService;
+
+    @BeforeClass
+    public static void init() {
+        executorService = Executors.newFixedThreadPool(NUM_OF_THREADS);
+        TimbermillServerOutputPipe pipe = new TimbermillServerOutputPipe.Builder().timbermillServerUrl("http://localhost:8484/events").build();
+        TimberLogger.bootstrap(pipe, TEST);
+    }
+
+    @Test
+    public void simpleStressTest() throws ExecutionException, InterruptedException {
+        Runnable simpleRunnable = () -> {
+            String taskId = runSimpleStress();
+            waitForEvent(taskId, Task.TaskStatus.SUCCESS);
+        };
+        runInParallel(simpleRunnable);
+    }
+
+    @Test
+    public void advanceStressTest() throws ExecutionException, InterruptedException {
+        Runnable advancedRunnable = () -> {
+            List<String> tasksIds = createAdvanceTasks();
+            String taskId = tasksIds.get(NUM_OF_TASKS - 1);
+            waitForEvent(taskId, Task.TaskStatus.SUCCESS);
+            String childTaskId = createChildrenTasks(tasksIds);
+            waitForEvent(childTaskId, Task.TaskStatus.SUCCESS);
+            Task childTask = client.getTaskById(childTaskId);
+            assertEquals(childTask.getCtx().get(CTX), childTask.getParentId());
+        };
+        runInParallel(advancedRunnable);
```

Comment:
the same comment as above, verification by a query that all tasks were written to elastic